must-gather/collection-scripts/gather_namespace_resources

```diff
@@ -20,3 +20,54 @@ for ns in $namespaces; do
     oc adm --dest-dir="${BASE_COLLECTION_PATH}" inspect installplan -n "${ns}" >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
 done
 
+# Resource List
+resources=()
+# collect storagecluster resources
+
+# TODO: Re enable the collection of storagecluster via inspect command
```

Comment:
A curiosity: What do we need to do to fix this?

Reply:
currently storagecluster yaml is collected by -oyaml.
in must-gather we want everything to be captured by inspect command.
when we use inspect command to collect storacluster resource it gives error.

Reply:
Do we know why?

Reply:
`oc adm inspect storagecluster`  starts collecting multiple resources e.g. noobaa/ (multiple resources gives error).  This needs to be fixed in oc repo.

Reply:
Weird! Okay, can we find or raise an issue with the oc repo and document it here? Doesn't have to be in this PR, I won't hold it up for that.

Reply:
ok.